app/views/layouts/application.html.erb.tt

```diff
@@ -25,6 +25,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+    <%%# Prevents the app is indexed by Google %>
+    <%% if ENV.fetch("NO_INDEX", "YES") == "YES" %>
```

Comment:
I like that this defaults to being on - that seems like a good default to me

Reply:
Yeah, default is blocking search engines if the env var is missing. Therefore, there is nothing to be changed